it/test/uk/gov/hmrc/alcoholdutyaccount/controllers/PaymentsIntegrationSpec.scala

```diff
@@ -0,0 +1,351 @@
+        |      "taxPeriodTo": "2024-02-29",
+        |      "businessPartner": "2315843349",
+        |      "contractAccountCategory": "51",
+        |      "contractAccount": "986963317689",
+        |      "contractObjectType": "ZADP",
+        |      "contractObject": "04194429852574946140",
+        |      "sapDocumentNumber": "764278200842",
+        |      "sapDocumentNumberItem": "0001",
+        |      "chargeReference": "XA63139412020838",
+        |      "mainTransaction": "6075",
+        |      "subTransaction": "6132",
+        |      "originalAmount": 20.56,
+        |      "outstandingAmount": 10.56,
+        |      "clearedAmount": 10.00,
+        |      "items": [
+        |        {
+        |          "subItem": "000",
+        |          "dueDate": "2024-02-01",
+        |          "amount": 10.56
+        |        },
+        |        {
+        |          "subItem": "001",
+        |          "dueDate": "2024-02-01",
+        |          "amount": 10.00,
+        |          "clearingDate": "2024-02-01",
+        |          "clearingReason": "Incoming Payment",
+        |          "clearingSAPDocument": "744695668171"
+        |        }
+        |      ]
+        |    },
+        |    {
+        |      "chargeType": "Payment on account",
+        |      "mainType": "On account",
+        |      "businessPartner": "3900513878",
+        |      "contractAccountCategory": "51",
+        |      "contractAccount": "907879960526",
+        |      "contractObjectType": "ZADP",
+        |      "contractObject": "26746239066398397458",
+        |      "sapDocumentNumber": "102204878592",
+        |      "sapDocumentNumberItem": "0001",
+        |      "mainTransaction": "0060",
+        |      "subTransaction": "6132",
+        |      "originalAmount": -1000,
+        |      "outstandingAmount": -1000,
+        |      "items": [
+        |        {
+        |          "subItem": "000",
+        |          "dueDate": "2024-08-01",
+        |          "amount": -1000
+        |        }
+        |      ]
+        |    },
+        |    {
+        |      "chargeType": "Payment on account",
+        |      "mainType": "On account",
+        |      "businessPartner": "7424508742",
+        |      "contractAccountCategory": "51",
+        |      "contractAccount": "082655887559",
+        |      "contractObjectType": "ZADP",
+        |      "contractObject": "50459162893080807444",
+        |      "sapDocumentNumber": "238469730497",
+        |      "sapDocumentNumberItem": "0001",
+        |      "mainTransaction": "0060",
+        |      "subTransaction": "6132",
+        |      "originalAmount": -1500,
+        |      "outstandingAmount": -500,
+        |      "clearedAmount": -1000,
+        |      "items": [
+        |        {
+        |          "subItem": "000",
+        |          "dueDate": "2024-08-01",
+        |          "amount": -500
+        |        },
+        |        {
+        |          "subItem": "001",
```

Comment:
This json is wrong, POA replaces, so would only have 1 item showing with original = outstanding = amount of -500 - and no clearedAmount